core/Constants/CoreConstants.cs

```diff
@@ -31,6 +31,11 @@ public static class CoreConstants
         /// </summary>
         public const string PackageJson = "package.json";
 
+        /// <summary>
+        /// The CMF package file name
```

Comment:
Maybe change `package` to manifest?

Reply:
In this case it is not the manifest yet, but cmfApp.json. Will keep it documented as CMF app file name